contrib/opencensus-ext-azure/opencensus/ext/azure/common/storage.py

```diff
@@ -170,3 +175,22 @@ def put(self, data, lease_period=0, silent=False):
             ),
         ))
         return blob.put(data, lease_period=lease_period, silent=silent)
+
+    def _check_storage_size(self):
+        size = 0
+        for dirpath, dirnames, filenames in os.walk(self.path):
+            for f in filenames:
+                fp = os.path.join(dirpath, f)
+                # skip if it is symbolic link
+                if not os.path.islink(fp):
+                    size += os.path.getsize(fp)
+                    if size >= self.max_size:
+                        logger.warning(
```

Comment:
I'm thinking this log could be bad here, the scenario where the storage size limit is reached is mostly because the machine cannot reach Azure endpoint for some reason and all telemetry is being rejected, if they are also sending log telemetry this could cause a perf impact depending on how often the telemetry is being sent, I can see other SDKs actually log the error so not a big deal just something to consider.

Reply:
Loggers use `name` to determine uniqueness. The logger used here is not the logger used in the user's application, it is the logger corresponding to `storage.py`, so no telemetry will ever be sent.